monero-outreach-docs/translations/es/storyboard/randomX_es.md

```diff
@@ -0,0 +1,35 @@
+ 
+¿Qué año es? Parece que solo fue ayer cuando decían que una moneda privada no podía reducir su tamaño de almacenamiento y que los ASICs ganarían al final. Pero aquí estamos, con los cascos espaciales puestos...reduciendo el tamaño del almacenamiento y hablando de RandomX. RandomX aún debe pasar por un proceso de prueba y autenticidad, pero es una parte emocionante del desarrollo de Monero. Esta guía da una descripción general de su naturaleza y plan, y, para que puedas aprender más. 
+
+## _¿Qué es RandomX?_  
+
+RandomX es un algoritmo de prueba-de-trabajo (PoW - siglas en inglés) nuevo que está programado para ser usado en la próxima actualización de Monero. RandomX está diseñado para ser resistente a los ASIC al utilizar códigos de ejecución aleatorios y técnicas de memoria exigentes para prevenir la dominación de la red por hardware de minería especializado. Siendo el proposito general de RandomX optimizar los CPUs, la distribución de la recompensa en la red será más descentralizada e igual para todos. 
+ 
+Howard Chu (hyc) estará hablando sobre RandomX en la conferencia de Monero, [Monero Konferenco](https://monerokon.com/), mientras tanto, visita el [repositorio en Github de RandomX](https://github.com/tevador/RandomX) para más detalles. Al momento de escribir este artículo, RandomX está empezando el proceso de ser auditado por [Trail Of Bits](https://www.trailofbits.com/), [X41](https://www.x41-dsec.de/), [Quarkslab](https://www.quarkslab.com/en/) y el [Grupo Kudelski](https://www.nagra.com/). RandomX está programado a lanzarse en la próxima actualización en la red de Monero.
+
+## _¿Qué va a cambiar?_ 
+
+Los ASICs serán los principales afectados por RandomX, y como será optimizado para los CPUs, los GPUs no tendrán el mismo incremento en el hashrate o en el poder de hashes. Referencias anteriores en las tarjetas de video [Nvidia (CUDA)](https://github.com/SChernykh/RandomX_CUDA) revelan un incremento del hashrate entre un 100% y un 150%, y se espera que se mejore con mayor optimización. Al ser RandomX exigente en memoria, se espera que los botnets y el malware en la minería se reduzca al ser el consumo de la memoria fácilmente detectado por los administradores. Una reducción en general del total del hashrate en la red incrementará la recompensa a los mineros legítimos, sin importar si minan con CPUs o GPUs. 
+
+RandomX tiene dos modalidades con diferentes requerimientos y rendimientos de memoria. Modo rápido requiere de 2 GB de memoria compartida pero tiene 4-6 veces más del rendimiento del Modo ligero que solo requiere de 256 MB de memoria RAM. El Modo rápido está destinado a mineros dedicados. El Modo ligero está diseñado para permitir a los nodos completos validar bloques sin necesitar las 2+GB de memoria RAM, para que los dispositivos pequeños (cómo los computadores ARM con board único, cómo por ejemplo, el Rock64) puedan seguir siendo usados como nodos independientes.
```

Comment:
+ _**El** Modo Rápido requiere de 2 GB...

+ (cómo los computadores ARM con board único, cómo por ejemplo, el Rock64): you may change the first _cómo_ to _como_, the same for the the second one, or either remove it of the sentence, any option would be correct.

Reply:
changed :)